### PR TITLE
[New] Implement Jumbotron Component

### PR DESCRIPTION
**Pull request by alphaoreo**
⭐ **New Features:**

- Implemented Jumbotron Component
- Added testing for Jumbotron Component

### CODE REVIEW

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: src/components/Jumbotron/Jumbotron.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export interface JumbotronProps {
+  /** The content of the jumbotron */
+  children?: React.ReactNode;
+  /** Determines if the jumbotron takes up the full width on the container with no rounded corners. Should be */
+  fluid?: boolean;
+}
+
+const Jumbotron = ({ children, fluid = false, ...rest }: JumbotronProps) => {
+  const style = fluid ? 'container-fluid' : '';
+  const finalStyle = `jumbotron ${style}`;
```
[the remark this answers is not in the document]
Comment (alphaoreo) [reply src/components/Jumbotron/Jumbotron.tsx:14]: Sorry I did not see this but it is done now.

**Review by Rizvida1 — APPROVED** [review]
Looks good!

**Review by dynamic11 — COMMENTED** [review]
put jumbotron doesn't seem to match the on the wet website. is this expected?

**Comment by alphaoreo** [comment]
> put jumbotron doesn't seem to match the on the wet website. is this expected?

Should I replace the div that makes a jumbotron with a section tag?

**Comment by dynamic11** [comment]
> > put jumbotron doesn't seem to match the on the wet website. is this expected?
> 
> Should I replace the div that makes a jumbotron with a section tag?

If they are using section then so should we 🔍

**Pull request merged**